tests/is/L42/tests/L42Source/TestHttpRequest/This.L42

```diff
@@ -52,23 +52,23 @@ Writer = {class method mut GW #$of(S that) =
 
 //Module 0  -- Towel AdamsTowel --> AdamsTowel
 TaskDeployAdams = DeployGit.towel(Trait({reuse [AdamsTowel]})
-  on=Url"github.com/Language42/is/AdamsTowel.L42"
+  on=Secret.adamstowelrepo()
   writer=Writer.#$of(S"AdamsTowel")
   )
 
 //Module 1
 TaskDeployHttpRequest = DeployGit.towel(HttpRequestDeployed()
-  on=Url"github.com/Language42/is/HttpRequest.L42"
+  on=Secret.httpreqrepo()
   writer=Writer.#$of(S"HttpRequest")
   )
 //Module 2
 TaskDeployGitWriterDeployed = DeployGit.towel(GitWriterDeployed()
-  on=Url"github.com/Language42/is/GitWriter.L42"
+  on=Secret.gitwriterrepo()
```

Comment:
This does not make a lot of sense,
I was sort of ok but sad about having Secret containing a 'repo()' that was containing the 
Url"github.com/Language42/is/"
Now we have a bunch of random getters, one for each deployed library? this does not work.
We can just have Secret.baseRepo()+"..."